arapp.json

```diff
@@ -13,5 +11,11 @@
       "params": []
     }
   ],
-  "path": "contracts/CounterApp.sol"
+  "path": "contracts/CounterApp.sol",
```

Comment:
Would put `path` last and `environments` first, to sort the keys by alphabetical order.

Reply:
Ok, do you think we should change this also on aragonCLI. Right now `aragon init` use this order